pkg/component/networking/vpn/seedserver/seedserver.go

```diff
@@ -619,36 +616,22 @@ func (v *vpnSeedServer) podTemplate(configMap *corev1.ConfigMap, secretCAVPN, se
 			},
 		}
 
-		if v.values.DisableNewVPN {
-			statusPath := filepath.Join(volumeMountPathStatusDir, "openvpn.status")
-			exporterContainer.Command = []string{
-				"/openvpn-exporter",
-				"-openvpn.status_paths",
-				statusPath,
-				"-web.listen-address",
-				fmt.Sprintf(":%d", metricsPort),
-			}
-			exporterContainer.Env = nil
-		}
-
 		template.Spec.Containers = append(template.Spec.Containers, exporterContainer)
 	}
 
-	if !v.values.DisableNewVPN {
-		template.Spec.InitContainers = []corev1.Container{
-			{
-				Name:            "setup",
-				Image:           v.values.ImageVPNSeedServer,
-				ImagePullPolicy: corev1.PullIfNotPresent,
-				Command: []string{
-					"/bin/vpn-server",
-					"setup",
-				},
-				SecurityContext: &corev1.SecurityContext{
-					Privileged: ptr.To(true),
-				},
+	template.Spec.InitContainers = []corev1.Container{
+		{
+			Name:            "setup",
+			Image:           v.values.ImageVPNSeedServer,
+			ImagePullPolicy: corev1.PullIfNotPresent,
+			Command: []string{
+				"/bin/vpn-server",
+				"setup",
 			},
-		}
+			SecurityContext: &corev1.SecurityContext{
+				Privileged: ptr.To(true),
+			},
+		},
 	}
```

Comment:
Let's move this block into the `template` initialisation as it is unconditionally executed anyway.